src/appshell/qml/Preferences/internal/ImportStyleSection.qml

```diff
@@ -41,7 +41,12 @@ BaseSection {
     QtObject {
         id: prv
 
-        property bool useStyleFile: root.styleFileImportPath !== ""
+        property bool useStyleFileExplicitlySet: root.styleFileImportPath !== ""
```

Comment:
These changes were needed to properly update the **Styles used for import** controls on the **Import** page and fix these two bugs:
        1. Open the **Preferences** dialog and navigate to the **Import** page.
        2. Change **Style used for import** from **Built-in style** to **Use style file**.
        3. Type a file if you want or skip this step.
        4. Click **Reset Preferences** without closing the dialog.
        **Result:** the **Use style file** radiobutton remains selected.
        
    AND:
        1. Open the **Preferences** dialog and navigate to the **Import** page.
        2. Change **Style used for import** from **Built-in style** to **Use style file**.
        3. Type somethinf in the file textbox.
        4. Delete the text in the file textbox.
        Result: the **Built-in style** radiobutton becomes selected.